website/docs/components/stepper/nav/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,87 @@
+## Usage
+
+### When to use
+
+- In complex flows that are broken up into multiple steps to more easily segment content.
+- To communicate what has and has not been completed in a multi-step flow.
+- For a single, on-page experience that requires completion.
+
+### When not to use 
+
+- For non-complex flows, use the [form pattern](/patterns/form-patterns) on a single page.
+- For simple forms or when progress doesn't need to be tracked, use structured page content.
+- For progression that doesn’t require a user to remain on the page for, use the [Stepper List](/components/stepper/list).
+
+## Interactive vs non-interactive
+
+The Stepper Nav allows for both interactive (default) and non-interactive variants. 
+
+### Interactive steps
+
+A user is able to navigate backwards within the Stepper Nav, however, they cannot navigate forward unless they have completed their current step. This creates consistency for step flow behavior and order of operations.
+
+![](/assets/components/stepper/navigation/stepper-navigation-interactive-steps.png)
+
+![The user's mouse is hovering over step one of four, showing its hover state.](/assets/components/stepper/navigation/stepper-navigation-interactive-backwards-movement.png)
+
+### Non-interactive steps
+
+The non-interactive variant is for read-only purposes and does not allow backwards navigation.
+
+![](/assets/components/stepper/navigation/stepper-navigation-non-interactive-steps.png)
+
+## Content
+
+The Stepper Nav has a title and an optional description. Content should be limited to necessary context, and long form content should live within the body of the page.
```

Comment:
[suggestion] Somewhat more formal, and removal of "context" which read a bit strange to me.

```suggestion
The Stepper Nav supports a title and an optional description. Content should be limited to only what is necessary, and long form content should live within the body of the page.
```